api/metric/api.go

```diff
@@ -244,16 +316,19 @@ func WithKeys(keys ...core.Key) OptionApplier {
 	}
 }
 
-// WithMonotonic sets whether a counter or a gauge is not permitted to
-// go down.
-func WithMonotonic(monotonic bool) CounterGaugeOptionApplier {
-	return counterGaugeOptionWrapper{
+// WithMonotonic sets whether a counter, a gauge or an observer is not
+// permitted to go down.
```

Comment:
May be we should note that this option still requires some work in the spec and should be used with caution.
For example, how will observer monotonicity will be enforced?

Reply:
This is for a future PR.